### ISSUE TITLE
CAR issue with probes 3B2 - sync channel 385

### ISSUE THREAD

**Issue by PierreLeMerre** [cortex-lab/neuropixels#27]
Hi,

I had a couple of issues with the "applyCARtoDat.m" script applied to the binary files obtained from the latest SpikeGLX (Release v.20190327) and the imec PXI card with neuropixel probes 3B2. 

First, a small error appears while loading "imec.ap.bin" due to the fact that the latest release of SpikeGLX change the file name to "imecX.ap.bin" where X is the probe port number on the imec PXI card. I guess this just need to be updated and can easily be solved by renaming the file as a temporary fix.

Second, the binary file contains 385 channels where channel number 385 is the sync channel acquired through the SMA connector. I think this channel shouldn't be included in  the CAR loop. It is not matching the current channel map file (found [here](https://github.com/cortex-lab/neuropixels)) and it creates error while reading the traces and the waveforms from the binary in Phy later on.

I fixed the second issue by creating CAR binary files with 384 channels by modifying the "applyCARtoDat.m" and saving it elsewhere as a temporay fix. 

Anybody having similar issues?

**Comment by nsteinme** [comment]
Yes, but since it takes the median of 385 channels, one bogus/noisy channel
does not make any difference in practice...
